ProjHttpsClient.ino

```diff
@@ -0,0 +1,161 @@
+/*********************************************************
+Project:              Solid enabled IoT sensor platform
+Team:                 INSA 5ISS Solid-IoT team
+Author:               Xiaotong XIE
+Function:             POST sensor data to Solid POD https://solid-iot.solidcommunity.net/iot/
+Fingerprints(SHA1):   99:F4:66:89:DE:98:3A:B4:7F:AC:79:62:84:D1:48:CA:FC:CB:22:B5
+State:                Pass-test
+*********************************************************/
+
+#include <ESP8266WiFi.h>
+#include <WiFiClientSecure.h>
+#include <time.h>
+
+#define BAUD_SPEED 115200
+#define PIN_SENSOR A0
+
+const char* ssid = "HUAWEI"; // WiFi name
+const char* password ="xiexiaotong"; // WiFi password
```

Comment:
Usually, you would prefer not to expose passwords on git, and only fill them in locally.

```suggestion
const char* password ="REPLACE ME"; // WiFi password
```